src/nvim/tui/tui.c

```diff
@@ -350,6 +352,13 @@ static void tui_resize(UI *ui, int width, int height)
   data->scroll_region.left = 0;
   data->scroll_region.right = width - 1;
   data->row = data->col = 0;
+  const char *term = os_getenv("TERM");
+  if (term && (STARTS_WITH(term, "xterm") || STARTS_WITH(term, "rxvt"))) {
+      // try to resize the terminal window
+      char r[16]; // enough for 9999x9999
+      sprintf(r, "\x1b[8;%d;%dt", height, width);
```

Comment:
What happens if we send this to a terminal that _doesn't_ support resizing? Is it just ignored? I'm wondering if we can skip the `STARTS_WITH` checks, which will turn into a mess as we keep adding checks all over the place...